pktoken/pktoken.go

```diff
@@ -36,9 +36,15 @@ type PKToken struct {
 	Cos     *Signature // Cosigner Signature
 }
 
-func New(idToken []byte, cicToken []byte) (*PKToken, error) {
+func New(idToken []byte, cicToken []byte, isGQ bool) (*PKToken, error) {
```

Comment:
What do you think about?
```golang
func New(idToken []byte, cicToken []byte, sigType SignatureType,) 
```

I like the property that this would document what is being flagged
``` 
pktoken.New(idToken.Bytes(), cicToken, pktoken.Gq)
```
vs a naked Boolean
``` 
pktoken.New(idToken.Bytes(), cicToken, false)
```

Just a idea, not a blocking request

Reply:
I have some small issues with the signature type and exposing it like that. Ideally, I would like to be able to rely on the `arg` value in the header to figure out whether or not it's a GQ signature and verify based on that instead of an unprotected value. So, for now I want to keep it a bool because that's what we have elsewhere and hopefully we won't need that in the future.